corehq/apps/app_manager/fields.py

```diff
@@ -160,8 +160,9 @@ class ApplicationDataRMIHelper(object):
     APP_TYPE_NONE = 'no_app'
     APP_TYPE_UNKNOWN = 'unknown'
 
-    def __init__(self, domain, as_dict=True, form_placeholders=None, case_placeholders=None):
+    def __init__(self, domain, user, as_dict=True, form_placeholders=None, case_placeholders=None):
```

Comment:
I checked and it looks like none of the other arguments are ever set directly. do you know why they exist?